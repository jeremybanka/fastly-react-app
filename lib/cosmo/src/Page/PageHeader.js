/* @flow */

import * as React from "react";
import { Box } from "../";

type Props = { children: React.Node };

const PageHeader = (props: Props) => (
  <Box paddingBottom="md" marginBottom="lg" borderBottom="regular">
    {props.children}
  </Box>
);

export default PageHeader;
