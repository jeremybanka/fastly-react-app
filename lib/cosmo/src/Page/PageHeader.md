The `<Page.Header />` component should be nested inside of a `<Page />`.
