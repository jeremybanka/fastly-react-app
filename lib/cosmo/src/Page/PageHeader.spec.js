/* @flow */

import React from "react";
import { shallow, render } from "enzyme";

import PageHeader from "./PageHeader";

describe("PageHeader test suite", () => {
  it("renders with className .PageHeader", () => {
    expect(shallow(<PageHeader />).is(".PageHeader")).toBe(true);
  });

  it("renders the stored snapshot", () => {
    expect(render(<PageHeader />)).toMatchSnapshot();
  });
});
