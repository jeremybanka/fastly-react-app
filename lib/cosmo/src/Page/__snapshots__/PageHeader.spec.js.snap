// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader test suite renders the stored snapshot 1`] = `
<div
  class="PageHeader"
/>
`;
